src/tests/t_kdb.py

```diff
@@ -117,6 +117,16 @@ def kldaputil(args, **kw):
 if not ldapmodify:
     skip_rest('some LDAP KDB tests', 'ldapmodify not found')
 
+ldapsearch = which('ldapsearch')
```

Comment:
Although it is very likely that ldapmodify and ldapsearch are either both present or both absent, we should explicitly check.  Put the ldapsearch assignment up next to the ldapmodify assignment, and change the conditional on skip_rest() to skip the rest of the script if either is not found.